locales/fr.json

```diff
@@ -279,6 +291,10 @@
       "pass": "Les attributs ARIA sont utilisés correctement pour le rôle défini",
       "fail": "{{=it.data && it.data.length > 1 ? 'Attributs ARIA non autorisés' : 'Attribut ARIA non autorisé'}} : {{~it.data:value}} {{=value}}{{~}}"
     },
+    "aria-allowed-role": {
+      "pass": "Le rôle ARIA role est autorisé pour l’élément donné",
```

Comment:
"role" appears twice: a translation of _Ensures role attribute has an appropriate value for the element_ could use the word attribute for one of them.  
`L'attribut ARIA role (…)` for example.